Dockerfile

```diff
@@ -24,4 +24,4 @@ RUN chmod +x /wait-for-services.sh
 
 EXPOSE 7777
 
-CMD ["/wait-for-services.sh", "cars-mysql", "3306", "cars-k3s", "6443", "npm", "run", "start:dev"]
```

Comment:
Shouldn't we still wait for services before letting the container come up?

Reply:
In my opinion, no. In docker compose world, `depends_on` allows us to control the startup order of the services which in this case `mysql` is expected to come up first (https://github.com/bitcoin-sv/cars-node/blob/master/docker-compose.yml#L45). If we want to add `k3s` to this list we can.

In a Kube world, I think it's better for the server to fail immediately if it can't connect to mysql, and the scheduler will restart the pod within reasonable restrictions.

Additionally, this Dockerfile `CMD` doesn't respect the environment variables a user can set for the mysql hostname; so it's easier if we remove it.